.github/workflows/ci-prover-e2e.yml

```diff
@@ -50,7 +50,8 @@ jobs:
             --base-token-price-nominator 1 \
             --base-token-price-denominator 1 \
             --set-as-default true \
-            --ignore-prerequisites 
+            --ignore-prerequisites \
+            --evm-emulator false
```

Comment:
Why are you changing this? It looks unrelated to making the CI mandatory.

Reply:
yes, but this is also the fix for the test itself... otherwise test won't pass and it won't be merged, I can add it in description if it's fine

Reply:
ok, sounds good